webui/src/components/App.css

```
@@ -489,6 +526,57 @@
+	gap: var(--space-xs) var(--space-lg);
+	font-family: var(--font-mono);
+	font-size: 0.85rem;
+}
+
+.existing-metadata dt {
+	color: var(--text-muted);
+}
+
+.existing-metadata dd {
+	color: var(--text-primary);
+	word-break: break-word;
```

Stylelint (17.14.1)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)